src/index.html

```diff
@@ -1,13 +1,62 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = ""
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
+  </head>
+  <body class="body">
+    <div class="card" data-qa="card">
+      <div class="card__image-container">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image-container-img"
+        >
+      </div>
+
+      <div class="card__full-product">
+        <div class="card__full-product-name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+
+        <div class="card__full-product-code">Код товара: 195434</div>
+      </div>
+      <div class="card__product-raiting">
+        <div class="card__product-raiting-stars-4">
```

Comment:
Use `stars` block with `stars--4` modifier the same way you did in previous task.
It should work correctly with `stars--3` modifier as well.